recipes/insilicoseq/run_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+curl -L https://ndownloader.figshare.com/files/15127220 > bogus-genome.fasta
+curl -L https://ndownloader.figshare.com/files/15127223 > checksums.sha
+
+iss generate --model MiSeq --seed 42 --cpus 1 --draft bogus-genome.fasta -n 100 -o bogus-reads
+shasum -c checksums.sha
```

Comment:
try `md5sum`, that might be more available

Reply:
Will try.

Reply:
Hallelujah that did it!